### DISCUSSION TITLE
如何仅关闭ctrl的翻译图片功能

### DISCUSSION THREAD

**Discussion by Yjun233** [immersive-translate/immersive-translate#2373] [Help] [answered]
### 概述 | Summary

我的ctrl键在图片上绑定到了另一个放大镜插件上，但现在放大图片时，总会弹出一个窗口让我订阅pro会员，对我的使用造成了不便，请问如何仅关闭ctrl的翻译图片功能，但保留翻译文本的功能？

**Comment by jasperyou** [comment]
没有遇到过, 你可以试试关闭以下两个选项来解决, 如果无法解决, 请补充更多详情, 比如截个git图等
1.Disable  Comic Floating Ball
chrome-extension://bpoadfkcbjbfhfodiogcnhhhpibjhbnh/options.html#manga
2. Disable Floating Button Ball
chrome-extension://bpoadfkcbjbfhfodiogcnhhhpibjhbnh/options.html#floating

**Comment by keyboard3** [comment]
https://dash.immersivetranslate.com/#abount 更新下规则，现在应该已经关闭了，再试试

**Comment by Yjun233** [comment]
现在正常了，谢谢

**Comment by Yjun233** [comment]
又遇到相同的问题了，bilibili和X均会出现
放大镜插件是imagus，绑定的ctrl键

**Reply by keyboard3** [reply, marked as answer]
https://dash.immersivetranslate.com/#developer
![image](https://github.com/user-attachments/assets/1cdb3992-d727-4910-bc36-e4502e40286f)

```
"imageRule.add": {
 "enableMouseHover": false
}

**Reply by Yjun233** [reply]
谢谢！
